packages/webpack-import-meta-loader/webpack-import-meta-loader.js

```diff
@@ -24,9 +24,34 @@ module.exports = function(source) {
   const browserPath = toBrowserPath(relativePath);
 
   const fileName = this.resource.substring(this.resource.lastIndexOf(path.sep) + 1);
-  return source.replace(
-    regex,
-    () =>
-      `({ url: \`\${window.location.protocol}//\${window.location.host}/${browserPath}/${fileName}\` })`,
-  );
+
+  let found = false;
+  let rewrittenSource = source.replace(regex, () => {
+    found = true;
+    return `({ url: getAbsoluteUrl('${browserPath}/${fileName}') })`;
+  });
+
+  if (found) {
+    return `
+      function getAbsoluteUrl(relativeUrl) {
+        const publicPath = __webpack_public_path__;
```

Comment:
Where is `__webpack_public_path__`? Is it available in a webpack module?

Reply:
Here is some more info on `__webpack_public_path__`: 

- https://wpack.io/concepts/how-publicpath-works/
- https://webpack.js.org/configuration/output/#outputpublicpath
- https://github.com/webpack/webpack/issues/2776#issuecomment-233208623
- https://webpack.js.org/api/module-variables/#__webpack_public_path__-webpack-specific

But basically, it is a variable known in any webpack module yes.